cmd/validate/image.go

```diff
@@ -40,20 +41,24 @@ type imageValidationFunc func(context.Context, string, policy.Policy, bool) (*ou
 
 func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 	var data = struct {
-		effectiveTime       string
-		filePath            string
-		imageRef            string
-		info                bool
-		input               string
-		output              []string
-		outputFile          string
-		policy              policy.Policy
-		policyConfiguration string
-		publicKey           string
-		rekorURL            string
-		snapshot            string
-		spec                *app.SnapshotSpec
-		strict              bool
+		certificateIdentity         string
+		certificateIdentityRegExp   string
+		certificateOIDCIssuer       string
+		certificateOIDCIssuerRegExp string
```

Comment:
Could be "certFoo" instead of "certificateFoo" here too perhaps. (Ps, I have nothing against the longer names if you like them, just something to consider.)

Reply:
Let's keep the long form for now. These are internal attributes that we could change later on without breaking anything.